scene/gui/option_button.cpp

```diff
@@ -394,6 +394,7 @@ void OptionButton::add_separator(const String &p_text) {
 void OptionButton::clear() {
 	popup->clear();
 	set_text("");
+	set_button_icon(nullptr);
```

Comment:
```suggestion
	set_button_icon(Ref<Texture2D>());
```
Prefer being explicit about typing